Source/WebInspectorUI/UserInterface/Views/ComputedStyleSection.js

```diff
@@ -221,6 +244,14 @@ WI.ComputedStyleSection = class ComputedStyleSection extends WI.View
             expandableView.element.classList.add("computed-property-item");
             this.element.append(expandableView.element);
         }
+
+        if (this._filterText || this._filterCleared)
+            this.dispatchEventToListeners(WI.ComputedStyleSection.Event.FilterApplied, {matches: !this.isEmpty});
+
+        if (this._filterCleared)
+            this._filterCleared = false;
+
+        this.dispatchEventToListeners(WI.ComputedStyleSection.Event.LayoutUpdated, {isEmpty: this.isEmpty})
```

Comment:
This seems kinda like a layering violation.  Can we use `didLayoutSubtree` instead?

Reply:
Good idea! Replaced dispatching an event with `didLayoutSubtree()`. Neat! I forgot about this.